tests/ui/macros/macro-metavar-expr-concat/syntax-errors.rs

```diff
@@ -113,4 +123,9 @@ fn main() {
     unsupported_literals!(_abc);
 
     empty!();
+
+    bad_literal_parameter!("\u{00BD}");
+    bad_literal_parameter!("\x41");
+    bad_literal_parameter!("🤷");
+    bad_literal_parameter!("d[-_-]b");
```

Comment:
Need test cases for `1`, `-1` and `true` (as non-string and multi-token literals).

Also need test cases not producing errors, and test cases passing a literal through `tt`.

Reply:
Done